features/step_definitions/file_steps.coffee

```diff
@@ -5,17 +5,21 @@ path = require 'path'
 module.exports = ->
 
   @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^I have a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
This quote from your professor is another indicator for how removed academics is from real-world software development.  Code is a lot more read than written (more than 10x), so optimizing for better readability isn't "wasting" time, its actually "saving" time. And fixing indentation is something that a good text editor does for you automatically. 

And even if not, those things can easily be tracked down and fixed in reviews. Indentation issues are not something that creates tough bugs that take hours to hunt down. Slight differences in code behavior, based on code that looks similar from a distance, and relies on easily overlooked boilerplate differences can cause such bugs easily though. That's especially true in JS-based languages that do zero argument verification. We need robust formatting conventions here, which provide us a margin of safety. 

I'd like to reserve left-aligned argment blocks for method calls with named arguments:

```coffeescript
createUser(
  firstname: 'Charlie',
  lastname: 'Rudolph')
```

calls the method with one argument that contains two keys. This is a very common style.

```coffeescript
createUser(
  'Charlie',
  'Rudolph')
```

calls the method with two arguments. These two forms are visually too close. People get used to expecting each argument on a separate line, and form habits. This can lead to really hard to track down bugs ("I'm getting the parameter, but somehow half of the data is missing!"). Consistently choosing the right-aligned form for the second case elegantly avoid this issue, through obvious visual separation. It allows developers to form helpful habits without running the risk of mixing completely different forms of argument styles accidentally. 

You'll probably reply "that's not a big deal to me, I can easily distinguish those cases", as you have to most points above. And yes, some exceptional developers can deal with all that well enough. But the coding conventions and styles shouldn't only work for the strongest links in the chain, but for all, including the weakest ones. Otherwise we set ourselves up for tons of mistakes when somebody new misses a comma or a parentheses, and somebody else approves this. We can't rely on Charlie having to find all those issues. This is not sustainable. 

I don't think there are any scientific answers here. This is merely a question of pragmatism, picking conventions wisely, based on decades of experience hunting down bugs in code bases with too ambiguous formatting.